apps/condo/domains/marketplace/gql.js

```diff
@@ -31,7 +31,7 @@ const MarketItemFile = generateGqlQueries('MarketItemFile', MARKET_ITEM_FILE_FIE
 const MARKET_ITEM_PRICE_FIELDS = `{ price { type group name price isMin vatPercent salesTaxPercent currencyCode } marketItem { id } ${COMMON_FIELDS} }`
 const MarketItemPrice = generateGqlQueries('MarketItemPrice', MARKET_ITEM_PRICE_FIELDS)
 
-const MARKET_PRICE_SCOPE_FIELDS = `{ marketItemPrice { id marketItem { id name sku organization { id } marketCategory { id name parentCategory { id name } } } price { type group name price isMin vatPercent salesTaxPercent currencyCode } } property { id } ${COMMON_FIELDS} }`
+const MARKET_PRICE_SCOPE_FIELDS = `{ marketItemPrice { id marketItem { id name sku organization { id } marketCategory { id name parentCategory { id name } } } price { type group name price isMin vatPercent salesTaxPercent currencyCode } } property { id address addressKey addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } map { ${PROPERTY_MAP_JSON_FIELDS} } } ${COMMON_FIELDS} }`
```

Comment:
Can we use `ADDRESS_META_SUBFIELDS_TABLE_LIST` instead of `ADDRESS_META_SUBFIELDS_QUERY_LIST` to request fewer fields?